### ISSUE TITLE
Poller Statistics Graphs are displayed in first column only

### ISSUE THREAD

**Issue by kozfelipe** [centreon/centreon#6003]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 2.8.17

**Centreon Engine version**: 1.8.1

**Centreon Broker version**: 3.0.11

**OS**: 2.6.32-696.16.1.el6.x86_64

**Steps to reproduce the issue:**
1. Home  >  Poller Statistics  >  Graphs

**Describe the results you received:**
all 9 Graphs are displayed in first column only

**Describe the results you expected:**
Poller Statistics Graphs should use 3 columns

**Additional information you think important (e.g. issue happens only occasionally):**

tested in 3 different browsers, no javascript errors

![image](https://user-images.githubusercontent.com/25208457/34740056-52d981e0-f565-11e7-955b-d2bfb6eec751.png)

**Comment by lpinsivy** [comment]
Hi,

What browser and version do you use?
What is the size of your screen?

**Comment by kozfelipe** [comment]
hi,

Tested in 1366-768 to 1920-1080
Chrome 63.0.3239.132 (64-bit)
Firefox 51.0 (64-bit)

- It works well in these browsers when I visit demo.centreon.com
- Resize window has no effects

**Comment by kozfelipe** [comment]
I found the error.

demo.centreon.com version of www/Themes/Centreon-2/style.css:

```
.graphZone {
    margin-bottom: 40px;
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    justify-content: space-between;
}
```

my style.css (same as 2.8.x branch):

```
.graphZone {
    margin-bottom: 40px;
    display: flex;
    flex-direction: column;
    flex-wrap: wrap;
    justify-content: space-between;
}
```

**Comment by lpinsivy** [comment]
No it is from latest version on github: https://github.com/centreon/centreon/blob/2.8.x/www/Themes/Centreon-2/style.css

**Comment by kozfelipe** [comment]
then latest version must be fixed in order to properly display columns

**Comment by Justice81** [comment]
Hello,

i had the same problem after an upgrade from 2.7.4 to 2.8.17
All graphs are displayed in only one row (Poller statistics and service graphs)

thanks for the help

**Comment by lpinsivy** [comment]
@Justice81 it will be corrected in next bugfix release

**Comment by kozfelipe** [comment]
it is now fixed on 2.8.18
